### ISSUE TITLE
Support stringifying BigInt

### ISSUE THREAD

**Issue by david-golightly-leapyear** [SebastianG77/true-json-bigint#3]
Repro:
```js
> const JSONBig = require('true-json-bigint')
undefined

> const o = { x: -1900719925474099n }
undefined
```

Expected:

```js
> JSONBig.stringify(o)
'{"x":-1900719925474099}'
```

Actual:
```js
> JSONBig.stringify(o)
'{}'
```

Note: `json-bigint` has the correct behavior

**Comment by SebastianG77** [comment]
hi @david-golightly-leapyear 

unfortunately I am quiet busy at the moment. So please give me a few days to investigate this issue.

Maybe this issue could be fixed in this branch as it contains the latest changes of json-bigint, but I have not checked yet by myself: https://github.com/SebastianG77/true-json-bigint/tree/25_parse_very_big_numeric_values

**Comment by SebastianG77** [comment]
Hi @david-golightly-leapyear,

I have merged this branch https://github.com/SebastianG77/true-json-bigint/tree/25_parse_very_big_numeric_values into master, such that now also Native BigInt is supported. Fixed version is 1.0.0. I have also already published to npm here:  https://www.npmjs.com/package/true-json-bigint

**Comment by david-golightly-leapyear** [comment]
@SebastianG77 thank you!
